docs/src/main/paradox/docs/delta/api/views/index.md

```diff
@@ -165,9 +154,52 @@ Request
 Response
 :   @@snip [view-list.json](../assets/views/list.json)
 
+## Indexing failures
+
+### Listing indexing failures
+
+This endpoint returns the indexing failures for the view `{view_id}`. The caller must have `view/write` permission on `{org_label}/{project_label}`.
+
+```
+GET /v1/views/{org_label}/{project_label}/{view_id}/failures
+    ?from={from}
+    &size={size}
+    &instant={instant}
+```
+
+#### Parameter description
+
+- `{from}`: Number - is the parameter that describes the offset for the current query; defaults to `0`
+- `{size}`: Number - is the parameter that limits the number of results; defaults to `20`
+- `{instant}`: Time Range - can be used to filter the results based on the moment they were risen
+
+**Example**
+
+Request
+:   @@snip [failures.sh](../assets/views/failures.sh)
+
+Response
+:   @@snip [failures.json](../assets/views/failures.json)
+
+### Fetch indexing failures as SSEs
+
+This endpoint fetches the available indexing failures. The `Last-Event-Id` is optional and provides the id of the indexing failure at which to start the stream; by default all indexing failures are fetched. The caller must have `view/write` permission on `{org_label}/{project_label}`.
+
+```
+GET /v1/views/{org_label}/{project_label}/{view_id}/failures/sse
+```
+
+**Example**
+
+Request
+:   @@snip [failures.sh](../assets/views/sse-failures.sh)
+
+Response
+:   @@snip [failures.json](../assets/views/sse-failures.json)
```

Comment:
I think that the content of the files (or the names) should be switched. The `sse-failures.json` contains what should be in `failures.json` and vice-versa.